server/services/gameCreate.js

```diff
@@ -3,9 +3,9 @@ const ValidationError = require('../errors/validation');
 const RANDOM_NAME_STRING = '[[[RANDOM]]]';
```

Comment:
Don't forget about the UI, we'll need to hide the stars per player and galaxy size settings if custom galaxy is selected.